src/sentry/static/sentry/app/views/events/searchBar.tsx

```diff
@@ -25,25 +26,34 @@ const FIELD_TAGS = Object.fromEntries(
   Object.keys(FIELDS).map(item => [item, {key: item, name: item}])
 );
 
-class SearchBar extends React.PureComponent {
-  static propTypes = {
+type SearchBarProps = Omit<React.ComponentProps<typeof SmartSearchBar>, 'tags'> & {
+  api: Client;
+  organization: Organization;
+  tags: {[key: string]: Tag};
```

Comment:
There is a `TagCollection` type in `app/stores/tagStore` that matches this type.

Reply:
TagCollection type is declared in multiple locations, as well as this type being hand coded, so I wasn't sure about pulling it

Reply:
If the types are the same we could/should pull the type into `app/types/index.tsx`